app/modules/ads/setup.js

```diff
@@ -144,6 +144,8 @@ export const adsSetup = {
 
     context.set('options.billTheLizard.cheshireCat', adsContext.opts.enableCheshireCat);
 
+    context.set('options.gamLazyLoading.enabled', isGeoEnabled('wgAdDriverGAMLazyLoadingCountries'));
```

Comment:
It's gonna be `appConfig.isGeoEnabled('wgAdDriverGAMLazyLoadingCountries')` after dev merge.